pylmod/gradebook.py

```diff
@@ -17,7 +17,25 @@
 
 
 class GradeBook(Base):
-    """API for functions that return gradebook data from MIT LMod service.
+    """
+    The MIT Learning Modules web service (LMod) always returns data.
+    Gradebook API calls return this data as a Python data structure,
+    either a list or a dictionary. The data structure will contain
+    these items:
+
+    - ``status`` - ``1`` = successful, ``-1`` = failed
+    - ``message`` - details about any error condition, or success message
+    - ``data`` - the returned data in a list or dictionary, if applicable
```

Comment:
I am under the impression that the LMod web service always returns the same structure that only varies in what the ``data`` item contains.  I will verify this.